CHANGELOG.rst

```diff
@@ -3,6 +3,42 @@ Infoblox.Nios_Modules Release Notes
 ===================================
 
 .. contents:: Topics
+
+v1.7.0
+======
+
+Release Summary
+---------------
+This release brings new modules for extensible attribute definition and managing DNS nameserver groups. 
+Additionally, it includes various enhancements, new features, and bug fixes aimed at improving the system's overall functionality and performance.
+
+Minor Changes
+-------------
+- Added support for the `use_for_ea_inheritance` parameter in Host Record to inherit EA from Host address.
+- Added support for the `use_dns_ea_inheritance` parameter in Host Record to inherit EA from associated zone.
+- Enabled IPv4 support for PXE server configuration in the Host Record module.
+- Introduced `use_logic_filter_rules` & `logic_filter_rules` support for both IPv4 and IPv6 network and network container.
+- Added IPv6 network container support for the `nios_next_network` lookup plugin.
+- Added `use_range` parameter to the nios_next_ip lookup plugin, enabling lookup for the next available IP from a network range.
+- Upgraded the base WAPI version to 2.12.3.
+- Improved handling of DHCP options in DHCP Range, Network, and Network Container.
+
+New Modules
```

Comment:
Lets have new modules section above minor changes

Reply:
As from previous pattern Major Changes are before New modules to continue the same pattern keeping the Minor change before new modules.